src/Compilers/Core/CodeAnalysisTest/MetadataReferences/MetadataReferenceTests.cs

```diff
@@ -20,6 +20,7 @@
 using Xunit;
 using CS = Microsoft.CodeAnalysis.CSharp;
 using static Roslyn.Test.Utilities.TestMetadata;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
```

Comment:
Sort? There are a few other files that seem out of order though, so maybe its not enforced at this level?